src/ch/blinkenlights/android/medialibrary/MediaMetadataExtractor.java

```diff
@@ -369,6 +369,14 @@ private void populateSelf(HashMap bastp) {
 			}
 		}
 
+		// If only one of (ARTIST, ALBUMARTIST) is present, populate the other
+		if (!containsKey(ARTIST) && containsKey(ALBUMARTIST)) {
+			put(ARTIST, get(ALBUMARTIST));
+		}
+		if (containsKey(ARTIST) && !containsKey(ALBUMARTIST)) {
+			put(ALBUMARTIST, get(ARTIST));
+		}
```

Comment:
note to self: we should trigger a library rescan after this change was submitted.